example/faultManagment/exampleFaultManagement.js

```diff
@@ -0,0 +1,84 @@
+/*****************************************************************************
```

Comment:
This is really an "Example Fault Source" plugin. Fault management itself is provided by the Fault Management Plugin.

Reply:
Updated.